### PR TITLE
NETOBSERV-101 - loki statusURL

### PR DESCRIPTION
**Pull request by jpinsonneau**
Following https://github.com/observatorium/api/pull/350 we will not have access to loki ready, config, metrics and build infos endpoints using the gateway.

This PR allow to configure a `statusURL` in the CRD and send it to the console plugin

Related PR: https://github.com/netobserv/network-observability-console-plugin/pull/186

### CODE REVIEW

File: api/v1alpha1/flowcollector_types.go [hunk 1]
```diff
@@ -300,9 +300,15 @@ type FlowCollectorLoki struct {
...
 	// QuerierURL specifies the address of the Loki querier service, in case it is different from the
 	// Loki ingester URL. If empty, the URL value will be used (assuming that the Loki ingester
-	// and querier are int he same host).
+	// and querier are in the same host).
 	QuerierURL string `json:"querierUrl,omitempty"`
 
+	//+kubebuilder:validation:optional
+	// StatusURL specifies the address of the Loki /ready /metrics /config endpoints, in case it is different from the
+	// Loki querier URL. If empty, the QuerierURL value will be used.
+	// This is usefull to show error messages and some context in the frontend
```
Comment (jotak) [inline api/v1alpha1/flowcollector_types.go:309]: typo: useful

File: api/v1alpha1/flowcollector_types.go [hunk 2]
```diff
@@ -300,9 +300,15 @@ type FlowCollectorLoki struct {
 	//+kubebuilder:validation:optional
 	// QuerierURL specifies the address of the Loki querier service, in case it is different from the
 	// Loki ingester URL. If empty, the URL value will be used (assuming that the Loki ingester
-	// and querier are int he same host).
+	// and querier are in the same host).
```
Comment (jotak) [inline api/v1alpha1/flowcollector_types.go:303]: nitpicking, but "same host" is not totally correct IMO, I would rephrase as "same server", no? (they can be on same host (node), but different servers / deployments)

**Review by jotak — APPROVED** [review]
almost lgtm, just a typo + small remark

**Comment by jotak** [comment]
btw: what would be the URL to set when using the loki-operator with the gateway? is it documented?

**Comment by jpinsonneau** [comment]
> btw: what would be the URL to set when using the loki-operator with the gateway? is it documented?

You need to point query frontend in that case. Check https://github.com/netobserv/documents/pull/22/files#diff-7f51acaa5430f57ab824677d12e76ad94983d7dd45c949b7cb1335991a019996R97
